requirements.txt

```diff
@@ -1,7 +1,20 @@
+alembic
+beautifulsoup4
+bunch
 Flask
+Flask-WTF
+itsdangerous
+Jinja2
```

Comment:
@pypingou I think we should generate <code> requirements.txt </code> using <code> pip freeze </code>  because it removes overhead on us to track dependencies manually.